test/asm/break.asm

```diff
@@ -0,0 +1,23 @@
+FOR V, 1, 100
+      PRINT "{d:V}"
+      IF V == 5
+          PRINT " stop! "
+          BREAK
+      ENDC
+      PRINT ", "
+ENDR
+      PRINTLN "done {d:V}"
+
+rept 1
+  break
+  ; skips invalid code
+  !@#$%
+elif: macro
+  invalid
+endr
+
```

Comment:
It's worth checking that the line number is correct here, and also that a lack of blank line after the `endr` is properly handled.

Reply:
Which line number correctness are you referring to here? And will removing the blank line 18 between `endr` and `rept 1` be sufficient?

Reply:
Added `println "OK"` right after the `endr`, no more blank line; it executes correctly.

Reply:
Use `WARN` instead, so that the line number gets printed too.

Reply:
Done.